### DISCUSSION TITLE
Javascript bookmarklets Blocked

### DISCUSSION THREAD

**Discussion by AcidClown** [dragon731012/-WORKING-bookmarklets-and-games#34] [Q&A]
Last year I used to use bookmarklets everyday they were so damn useful but now this year when I click on any bookmarklet it just doesn't load so I did some research and discovered that my school blocked any url that starts with "javascript:"

If anyone can find anything it would be very very nice.

**Comment by Mazedotexe** [comment]
@AcidClown this is assuming your school only allows a few extensions. (it would be much simpler if it allowed all but if your school blocked bookmarklets, i doubt you can use all extensions.) check your extension allowlist ( chrome://policy and search extensioninstalla) and check out all the extensions ids. they will be things like "haldlgldplgnggkjaafhelgiaglafanh" (goguardian) and stuff like that. if youre lucky, they might allow some things like grammarly. you can check what extension an id is by going to
`https://chrome.google.com/webstore/detail/extensionidhere`
. hopefully, one extension allows you to manipulate javascript somehow. bookmarklets are blocked for me too but they allow an extension called 'shortkeys' (logpjaacgmcbpdkdchjiaagddngobkck) where you can set up keyboard shortcuts to do things. one of the things is running javascript, so its basically a keyboard shortcut runs a bookmarklet. if you can use shortkeys too, here is a nice text file you can input (what i use):
























[bookshortlets-idk.txt](https://github.com/dragon731012/-WORKING-bookmarklets-and-games/files/12799596/bookshortlets-idk.txt)





my favs are mince-raft and asteroids =)

**Reply by MrBrain295** [reply]
Do u know of any other apps that work for this?

**Comment by Mazedotexe** [comment]
> Do u know of any other apps that work for this?

not really, but check your policies! all you have to do is search for "ext" and you'll see extension allowlist. click show more to see the array and copy all of it. it should look something like this:
`[
  "aapbdbdomjkkjkaonfhkkikfgjllcleb",
  "cceilgmnkeijahkehfcgfalepihfbcag",
  "gdicnpbaekbefjanokchpfhnaphfnphl",
  "haldlgldplgnggkjaafhelgiaglafanh"
]`
the above is a small portion of my extension allowlist. then, you can paste it into [here](https://sites.google.com/view/extension-allowlist) (which i made just to answer this question) and find all the extensions your school allows. you might find something that can run js, you might not. either way, i hope this is a helpful tip.
